addon/components/nypr-player-integration.js

```diff
@@ -59,7 +59,7 @@ export default Component.extend({
   streamScheduleUrl    : reads('currentAudio.scheduleUrl'),
   streamPlaylistUrl    : computed('currentAudio.playlistUrl', function() {
     if (get(this,'currentAudio.playlistUrl')) {
-      return `/streams/${get(this, 'currentAudio.id')}`;
```

Comment:
If we're skipping over this view, can we also remove the route?

Reply:
Yes, but that would be done in the web clients.

Reply:
Ok, pending approval from K Stat we agreed to redirect the streams route to the full playlist page in both the wnyc and wqxr web clients.